crates/autopilot/src/run_loop.rs

```diff
@@ -221,12 +213,10 @@ impl RunLoop {
                 .orders
                 .iter()
                 .map(|order| {
-                    let (class, surplus_fee) = match order.metadata.class {
-                        OrderClass::Market => (Class::Market, None),
-                        OrderClass::Liquidity => (Class::Liquidity, None),
-                        OrderClass::Limit(LimitOrderClass { surplus_fee, .. }) => {
-                            (Class::Limit, surplus_fee)
-                        }
+                    let class = match order.metadata.class {
+                        OrderClass::Market => Class::Market,
+                        OrderClass::Liquidity => Class::Liquidity,
+                        OrderClass::Limit(_) => Class::Limit,
```

Comment:
To me, it looks like we can change the `enum OrderClass` to not need `LimitOrderClass` anymore.

Reply:
Will be done in a follow up. Leaving unresolved.